src/Network/Email/Email.php

```diff
@@ -1241,6 +1241,16 @@ public static function configTransport($key, $config = null)
     }
 
     /**
+     * Returns an array containing the named transport configurations
+     *
+     * @return array Array of configurations.
+     */
+    public static function configuredTransport()
```

Comment:
That is to say: I understand the desire to change it, but boy is it going to mean a lot more custom code in my plugin to handle this _one_ case. I'm fighting out of laziness at the moment.